booking.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Booking</title>
+    <link rel="stylesheet" href="static/css/main.css" />
+    <script type="module" src="src\JS\booking.js" defer></script>
+  </head>
+  <body>
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
```

Comment:
Would be great if you could change " L " in logo.png to uppercase. Think thats what creating problems when trying to load logo on github pages

Reply:
Or change the image to lowercase? what do you think?

Reply:
That works to!